src/lexer.coffee

```diff
@@ -1252,7 +1252,15 @@ REGEX = /// ^
 REGEX_FLAGS  = /^\w*/
 VALID_FLAGS  = /^(?!.*(.).*\1)[imguy]*$/
 
-HEREGEX      = /// ^(?: [^\\/#] | \\[\s\S] | /(?!//) | \#(?!\{) )* ///
+HEREGEX      = /// ^
+      (?:
+        [^\\/#]   |
+        \\[\s\S]  |
+        # The comment should consume everything until the end of the line, including the '///'
+        \#(?!\{)(?:[^\n\r]*[\n\r])  |
+        /(?!//)
```

Comment:
Any particular reason you changed the order, moving this case last?

Reply:
Actually no. It's a leftover from playing with the RegExp rules.